app/models/course.rb

```diff
@@ -284,10 +284,6 @@ def tracked_revisions
              .where(wiki_id: wiki_ids)
   end
 
-  def tracked_namespaces
```

Comment:
Rather than removing this, I think it should be replaced with an implementation that tries to return an array based on the course_wiki_namespaces and falls back to `[0]` if there are none. We want to allow for tracking an alternative namespace _instead of_ mainspace.

Reply:
Oh...so you mean we won't be storing all mainspace articles by default unless user has no tracked namespaces ?

Reply:
I thought the default behavior of including all the mainspace articles was still there...

Reply:
I think that would be best.